projects/1.19/assets/dimensional-doors/dimdoors/lang/zh_cn.json

```
@@ -0,0 +1,426 @@
+  "item.dimdoors.unstable_dimensional_door.info": "警告：它将把你传到一个随机的位置",
+  "block.dimdoors.oak_dimensional_door.info0": "放置在裂痕核心下方的方块",
+  "block.dimdoors.oak_dimensional_door.info1": "以进入裂痕对应的地方，或放置在地牢维度的",
+  "block.dimdoors.oak_dimensional_door.info2": "任何地方以离开地牢维度",
+  "item.dimdoors.creepy_record": "音乐唱片",
+  "item.dimdoors.creepy_record.desc": "Creepy - The sound of Limbo",
+  "item.dimdoors.eternal_fluid_bucket": "永恒源质桶",
+  "item.dimdoors.white_void_record": "音乐唱片",
+  "item.dimdoors.white_void_record.desc": "White Void - The sound of Pocket",
+  "item.dimdoors.dimensional_eraser": "空间抹除器",
+  "item.dimdoors.dimensional_eraser.desc": "瞬间擦除生物，不会掉落任何物品§8§n轰炸空间！",
+  "item.dimdoors.monolith_spawner": "监牢之瞳刷怪蛋",
+  "item.dimdoors.mask_wand": "Mask Wand",
+  "item.dimdoors.mask_shard": "Mask Shard",
+
+  "dimdoors.virtualTarget.dimdoors.available_link": "随机",
+  "dimdoors.virtualTarget.dimdoors.escape": "逃离",
+  "dimdoors.virtualTarget.dimdoors.global": "全局",
+  "dimdoors.virtualTarget.dimdoors.limbo": "维度监牢",
+  "dimdoors.virtualTarget.dimdoors.local": "本地",
+  "dimdoors.virtualTarget.dimdoors.public_pocket": "公用口袋维度",
+  "dimdoors.virtualTarget.dimdoors.pocket_entrance": "公用口袋维度入口",
+  "dimdoors.virtualTarget.dimdoors.pocket_exit": "公用口袋维度出口",
+  "dimdoors.virtualTarget.dimdoors.private": "私人口袋维度",
+  "dimdoors.virtualTarget.dimdoors.private_pocket_exit": "私人口袋维度出口",
+  "dimdoors.virtualTarget.dimdoors.relative": "相对",
+  "dimdoors.virtualTarget.dimdoors.id_marker": "Id标记",
+  "dimdoors.virtualTarget.dimdoors.unstable": "不稳定",
+  "dimdoors.virtualTarget.dimdoors.none": "未绑定",
+
+  "fluid.dimdoors.eternal_fabric": "永恒之壁",
+
+  "entity.dimdoors.monolith": "监牢之瞳",
+
+  "commands.dimteleport.usage": "/dimteleport <维度> <x> <y> <z> [左右角度] [上下角度]",
+  "commands.fabricconvert.usage": "/fabricconvert",
+  "commands.fabricconvert.success": "所有现实之壁已转变为黑色",
+  "commands.pocket.usage": "/pocket <组> <名称> [配置]",
+  "commands.pocket.group_not_found": "未找到名为%s的组",
+  "commands.dimdoors.pocket.template_not_found": "未找到名为%s的模板",
+  "commands.dimdoors.saveschem.usage": "/saveschem <名称>",
+  "commands.dimdoors.saveschem.success": "口袋维度%s已被成功保存",
+  "commands.generic.dimdoors.not_in_pocket_dim": "你只能在口袋维度维度使用此命令",
+  "commands.generic.dimdoors.not_in_pocket": "你只能在口袋维度中使用此命令",
+  "commands.generic.unknownValue": "错误：未知的值'%s'",
+  "commands.pocket.unknownPocketTemplate": "错误：未知的模板'%s'",
+  "commands.pocket.placedSchem": "已放置口袋维度模板%s于%s位置，维度%s",
+  "commands.pocket.loadedSchem": "已加载口袋维度模板%s至剪贴板，使用//paste粘贴",
+  "commands.pocket.log.creation.off": "已禁用口袋维度创建日志.",
+  "commands.pocket.log.creation.on": "已启用口袋维度创建日志",
+  "commands.pocket.log.creation.generating": "从'%s'模板生成口袋维度于位置 %s %s %s",
+
+  "rifts.unlinked1": "这条裂痕没有连接任何地方",
+  "rifts.unlinked2": "这条裂痕已经关闭",
+  "rifts.isLocked": "这条裂痕已经锁定",
+  "rifts.cantUnlock": "无法解锁此维度门！",
+  "rifts.unlocked": "解锁",
+  "rifts.locked": "锁定",
+  "rifts.destinations.escape.cannot_escape_limbo": "很好的尝试，但你必须找到永恒源质才能逃离维度监牢",
```

```suggestion
  "rifts.destinations.escape.cannot_escape_limbo": "想的挺不错，但你必须找到永恒源质才能逃离维度监牢",
```
`Nice try`实际是一种对失败情况的反讽